src/style.css

```diff
@@ -0,0 +1,98 @@
+.hello {
+  color: red;
+}
+
+.container_0 {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 10rem;
+}
+
+.container_1 {
+  width: 50%;
+  height: 500px;
+  border: 5px solid #c3c3c3;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.col1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.col1_1 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 5rem 0 5rem;
+}
+
+.hed0 {
+  margin-left: 2rem;
+}
+
+.hed2 {
+  margin-left: 5rem;
+}
+
+.btn1 {
+  height: 20px;
+  width: 70px;
+
+  /* margin-right:5rem; */
```

Comment:
- [ ] Please remove this commented line of code because I believe it is bad practice to leave comment code like this in your codebase and it makes your code look unclean.🤓 🛠️

Reply:
@ridwanediallo Thank you 👍 It is fixed.